### PR TITLE
PMM-12462 Add HA usage telemetry

### PR DESCRIPTION
**Pull request by ademidoff**
PMM-12462

Link to the Feature Build: SUBMODULES-3417

Note: this PR is a continuation of https://github.com/percona/pmm/pull/2532.
It adds the configuration to `managed/services/telemetry/config.default.yml`.

### CODE REVIEW

File: managed/services/telemetry/config.default.yml [hunk 1]
```diff
@@ -936,3 +936,38 @@ telemetry:
...
+      - metric_name: "pmm_server_ha_pmm_passive_ip"
+        column: "PMM_PASSIVE_IP"
+      - metric_name: "pmm_server_ha_pmm_passive_node_id"
+        column: "PMM_PASSIVE_NODE_ID"
+      - metric_name: "pmm_server_ha_pmm_passive2_ip"
+        column: "PMM_PASSIVE2_IP"
+      - metric_name: "pmm_server_ha_pmm_passive2_node_id"
+        column: "PMM_PASSIVE2_NODE_ID"
+      - metric_name: "pmm_server_ha_docker_image"
+        column: "DOCKER_IMAGE"
```
Comment (BupycHuk) [inline managed/services/telemetry/config.default.yml:973]: env vars should be taken from this block https://www.notion.so/percona/Setup-PMM-in-HA-mode-DRAFT-11fd2dca015d4f288a554ce02b7ac882?pvs=4#0ea56a2fd9044dfb80a77d7d6bc18e7f

Comment (ademidoff) [reply managed/services/telemetry/config.default.yml]: I was reflecting on whether we really need all of the variables listed there? It seems like all we need to know is whether or not `PERCONA_TEST_HA_ENABLE` is set to 1. The rest of the envvars is not relevant for our stats.

Well, maybe we need a couple more, but certainly not the IPs. :)

What do you think?

Anyway, I updated the list to reflect the vars that make sense from my standpoint. Please review.

File: managed/services/telemetry/config.default.yml [hunk 2]
```diff
@@ -937,35 +937,28 @@ telemetry:
...
+      - metric_name: "pmm_server_ha_enable"
+        column: "PERCONA_TEST_HA_ENABLE"
+      - metric_name: "pmm_server_ha_bootstrap"
+        column: "PERCONA_TEST_HA_BOOTSTRAP"
+
+  - id: PMMServerHAAdditional
+    source: ENV_VARS
+    transform:
+      type: StripValues
+    summary: "Use of HA feature, continued"
```
Comment (BupycHuk) [inline managed/services/telemetry/config.default.yml:957]:
```suggestion
    summary: "Use of External DBs"
```

File: managed/services/telemetry/config.default.yml [hunk 3]
```diff
@@ -937,35 +937,28 @@ telemetry:
...
+    summary: "Use of HA feature"
+    data:
+      - metric_name: "pmm_server_disable_builtin_clickhouse"
+        column: "PERCONA_TEST_PMM_DISABLE_BUILTIN_CLICKHOUSE"
+      - metric_name: "pmm_server_disable_builtin_postgres"
+        column: "PERCONA_TEST_PMM_DISABLE_BUILTIN_POSTGRES"
+      - metric_name: "pmm_server_ha_enable"
+        column: "PERCONA_TEST_HA_ENABLE"
+      - metric_name: "pmm_server_ha_bootstrap"
+        column: "PERCONA_TEST_HA_BOOTSTRAP"
```
Comment (BupycHuk) [inline managed/services/telemetry/config.default.yml:951]: I think we need only PERCONA_TEST_HA_ENABLE here

**Review by BupycHuk — APPROVED** [review]

File: managed/services/telemetry/config.default.yml (hunk 3 above)
Comment (ademidoff) [reply managed/services/telemetry/config.default.yml]: OK, done!

**Review by BupycHuk — APPROVED** [review]

**Review by JiriCtvrtka — APPROVED** [review]

**Pull request merged**